test/integration/ReadWriteTest.cc

```diff
@@ -18,8 +18,36 @@ INITIALIZE_EASYLOGGINGPP
 
 using asio::ip::tcp;
 using client_namenode_translator::ClientNamenodeTranslator;
+
+static const int NUM_DATANODES = 3;
+
 int num_threads = 4;
 int max_xmits = 2;
+int32_t xferPort = 50010;
+int32_t ipcPort = 50020;
```

Comment:
`const`?

Reply:
Ah, I missed the issue below. Can we add a comment here saying that it will be incremented?